warehouse/models/rt_staging/_rt_staging.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+
+sources:
```

Comment:
Can we make a `_src_rt.yml` [like I did for Airtable](https://github.com/cal-itp/data-infra/blob/extract-transit-tech-stacks-airtable/warehouse/models/staging/transit_database/_src_airtable.yml)? (I am trying to move more towards the actual dbt defaults as I do new work that's not just conversion from Airflow)

Reply:
yeah sure, I have no opinion either way